x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/post_comment.ts

```diff
@@ -161,6 +162,24 @@ export default ({ getService }: FtrProviderContext): void => {
     });
 
     describe('unhappy path', () => {
+      it('400s when attaching a file with metadata that is missing the file field', async () => {
+        const postedCase = await createCase(supertest, getPostCaseRequest());
+
+        await createComment({
+          supertest,
+          caseId: postedCase.id,
+          params: getFilesAttachmentReq({
+            externalReferenceMetadata: {
```

Comment:
You could use `fileAttachmentMetadata` defined in `x-pack/test/cases_api_integration/common/lib/mock.ts`.